stable/insights-agent/templates/kube-bench/serviceaccount.yaml

```diff
@@ -0,0 +1,8 @@
+{{- if .Values.kubebench.enabled -}}
```

Comment:
Whatever is decided with kubesec below would apply here too.

Reply:
I agree - we can probably drop the SA here (as well as `serviceAccountName` in the podspec)